lighthouse-viewer/test/drag-and-drop-test.js

```diff
@@ -17,8 +15,12 @@ testHelpers.setupJsDomGlobals();
 
 const DragAndDrop = require('../app/src/drag-and-drop.js');
 
-function assertUIReset() {
-  assert.ok(!document.querySelector('.drop_zone').classList.contains('dropping'));
+function createCustomEvent(type, key, value) {
+  const customDropEvent = new window.CustomEvent(type);
+  if (key && value) {
```

Comment:
right now it's a bit odd that the callers do `createCustomEvent('drag', 'dataTransfer', null)` when it seems like that doesn't really do anything, is this intentional?

Reply:
I think removing this fn entirely would be better.

```
// create custom drop event with mock files in dataTransfer
const event = new CustomEvent('drop');
event.dataTransfer = {
  files: ['mock file'],
};
document.dispatchEvent(event);
```

Reply:
Oh yeah ignore what I was saying, let's go with @connorjclark !